tests/testthat/test_paramtest_randomPlantedForest_regr_rpf.R

```diff
@@ -0,0 +1,31 @@
+test_that("regr.rpf train", {
+  learner = lrn("regr.rpf")
+  fun = randomPlantedForest:::rpf.data.frame
+  exclude = c(
+    "x", "y", # handled internally
+    "loss", # Not applicable for regression learner
+    "delta", # Not applicable for regression learner
+    "epsilon", # Not applicable for regression learner
+    "max_interaction_ratio",
```

Comment:
explain why excluded

Reply:
Added comments to explain that delta, epsilon are not applicable in regression, and loss can only have one value in that case